released/terraform-be/backendask8s.tf

```
@@ -0,0 +1,7 @@
+terraform {
+  backend "kubernetes" {
+    secret_suffix    = "state"
+    config_path      = "path/to/your/kubeconfig"
+    namespace        = "namespace"
```

I would make the name of the `namespace` more explicit e.g. state-storage
```suggestion
    namespace        = "state-storage"
```